File: programmers/lev1/12922.py
def solution(n):
    answer = ''
    wm=["수","박"]
    
    cnt=0

    while cnt<int(n):
        if cnt%2==0:
            answer+=wm[0]
        else:
            answer+=wm[1]
        cnt+=1
    
    print(answer)


    return answer

n="4"
solution(n)

File: java/src/programmers/year2023/lv1/_12922.java
package programmers.year2023.lv1;

public class _12922 {
    public static void main(String[] args) {

    }

    public static String solution(int n) {
        String[] arr = {"수", "박"};
        String answer = "";

        for (int i = 0; i < n; i++) {
            answer += arr[i % 2];
        }
        return answer;
    }
}
